backend/src/zango/core/utils.py

```diff
@@ -84,7 +87,10 @@ def get_current_request_url(request, domain=None):
 
 
 def get_datetime_in_tenant_timezone(datetime_val, tenant):
-    tz = pytz.timezone(tenant.timezone)
+    timezone = tenant.timezone
+    if not tenant.timezone:
+        timezone = tzlocal.get_localzone_name()
+    tz = pytz.timezone(timezone)
```

Comment:
@shahharsh176 should this behaviour be applied to everything that uses this method or only auditlogs